javascript/javascript3/week2/Tasks/script.js

```diff
@@ -0,0 +1,41 @@
+
+//Promise that resolves after set time
+function resolveAfterSeconds(resolveAfter){
+    return new Promise((resolve)=>{
+        setTimeout(()=>{
+            resolve();
+        },resolveAfter*1000)
+    })
+}
+
+resolveAfterSeconds(3).then(()=>{
+    console.log("Resolved");
+});
+
+//Fetching and waiting
```

Comment:
Both the Promise and the Async version miss the step of waiting 3 seconds before doing the fetch call.